NextCenturyCorporation/AIDA-Interchange-Format
# Adding m18 submission checker script

Linked to ticket AIDA-871

### Description:
This is designed to be a standalone python3 script that can be sent out to performers so that they can run it on their archives and check if their archive is submit ready. Submit ready means that the archive is structured properly for the AIF M18 batch validation process. 

#### To test: 

- Ensure that the task type was determined correctly; 
- that the output is clear as to which task type the archive was checked for; 
- that the program does not raise false positives in where it states an archive is submit ready when it should not be. 

#### To run:
`python3 aida_m18_submission_checker.py <path/to/archive>`
